atlasdb-processors-tests/build.gradle

```diff
@@ -0,0 +1,12 @@
+apply from: "../gradle/publish-jars.gradle"
+apply from: "../gradle/shared.gradle"
+
+apply plugin: 'org.inferred.processors'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    processor project(":atlasdb-processors")
```

Comment:
For some reason, I couldn't run the processor in the same package as it's source code. Maybe it's possible, but I didn't figure a way to do it.